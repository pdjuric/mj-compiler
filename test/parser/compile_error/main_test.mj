program main_test
{
    int main() { }
    void main(int a) { }
}