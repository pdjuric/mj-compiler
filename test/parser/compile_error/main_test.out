ERROR 17:07:19,461 - line   4 - Identifier main already defined in this scope.
ERROR 17:07:19,461 - line   1 - Function void main() is not defined.